pandas/tests/frame/test_axis_select_reindex.py

```diff
@@ -1151,3 +1151,24 @@ def test_raise_on_drop_duplicate_index(self, actual):
         expected_no_err = actual.T.drop('c', axis=1, level=level,
                                         errors='ignore')
         assert_frame_equal(expected_no_err.T, actual)
+
+    @pytest.mark.parametrize('index, drop_labels', [
+        ([1, 2, 3], []),
+        ([1, 1, 2], []),
+        ([1, 2, 3], [2]),
+        ([1, 1, 3], [1]),
+    ])
+    def test_drop_empty_list(self, index, drop_labels):
+        # GH 21494
+        expected_index = [i for i in index if i not in drop_labels]
+        frame = pd.DataFrame(index=index).drop(drop_labels)
+        tm.assert_frame_equal(frame, pd.DataFrame(index=expected_index))
+
+    @pytest.mark.parametrize('index, drop_labels', [
+        ([1, 2, 3], [1, 4]),
+        ([1, 2, 2], [1, 4]),
+    ])
```

Comment:
Similarly as above, I don't think you need to parametrize ``drop_labels``.

@jreback , what do you think? Do we have a general rule concerning making parameter combinations more explicit/more compact?